plugins/bufbuild/connect-es/v0.7.0/buf.plugin.yaml

```diff
@@ -0,0 +1,19 @@
+version: v1
+name: buf.build/bufbuild/connect-es
+plugin_version: v0.7.0
+source_url: https://github.com/bufbuild/connect-es
+description: Generates code that is compatible with browsers and Node.js for working with Connect, gRPC, and gRPC-Web.
+deps:
+  - plugin: buf.build/bufbuild/connect-es:v1.0.0
```

Comment:
I think the base types should still come from `buf.build/bufbuild/es:v1.0.0` ?

Reply:
Ah good catch.  Thank you.